packages/leap/lib/src/leap_options.dart

```diff
@@ -0,0 +1,53 @@
+/// A configurable class that allows the developer to
+/// customize different options that Leap will use
+/// when reading the map.
+class LeapConfiguration {
+  /// The tiled options, change it to configure how Leap
+  /// interpret the tiled map.
+  static TiledOptions tiled = const TiledOptions();
```

Comment:
I would slightly prefer this to be non-static, which I think means it has to be a field on LeapGame. Nothing should really be static in case there are multiple games or something

Reply:
Sounds good, I went the static route as I thought It would make it easier since we would not need to pass the object down in the classes tree. But it wasn't that hard in the end. Let me know if this is what you think

Reply:
Hm, I still think it would be better to set it on the `LeapGame` because the name implies it's config for the whole of Leap, which isn't just the Map. And you can always access `this.game` because `HasGameRef` is everywhere. Up to you though, looks great overall

Reply:
Oh, yes! That makes more sense to put on the game instead of the map! Sounds good, I will make the change sometime this weekend

Reply:
@kurtome I have changed so the game now holds the configuration, and passes it down to the other classes. The only thing is that I still kept LeapMap and other classes to hold an instance of `TiledOptions` instead of accessing it through the `gameRef` because these classes also need to access the options in their constructor body, and the `gameRef` is not set there.

I believe we could do those same operations that are being do in the constructor body, in the onLoad method, which would allow us to access the `gameRef`. But I thought on avoiding a bigger change than needed. So my proposal would be to land this as is, and we could revisit later, wdyt?

Reply:
LGTM